app/src/main/java/io/appium/uiautomator2/model/api/touch/PackageModel.java

```diff
@@ -0,0 +1,31 @@
+package io.appium.uiautomator2.model.api.touch;
+
+public class PackageModel {
+    String PackageName;
+    String PackageActivity;
+    String AppName;
+
+    public String getAppName() {
+        return AppName;
+    }
+
+    public String getPackageName() {
```

Comment:
Can any of the values be `null`? If yes then mark corresponding getters as `@Nullable`